### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by girarda**
## What
* The schema and configured schema files provided for the tutorial was deleted as part of [this commit](https://github.com/airbytehq/airbyte/commit/afb4a00bff7066d3349c6e54e6f0b0431dee1969)

## How
* Upload the schema to the docs directory

## Recommended reading order
1. `docs/connector-development/tutorials/cdk-tutorial-python-http/configured_catalog.json`
2. `docs/connector-development/tutorials/cdk-tutorial-python-http/exchange_rates_schema.json`
3. `docs/connector-development/config-based/tutorial/4-reading-data.md`
4. `docs/connector-development/tutorials/cdk-tutorial-python-http/declare-schema.md`
5. `docs/connector-development/tutorials/cdk-tutorial-python-http/read-data.md`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

### CODE REVIEW

File: docs/connector-development/config-based/tutorial/4-reading-data.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ See the [catalog guide](https://docs.airbyte.com/understanding-airbyte/beginners
 
 Let's define the stream schema in `source-exchange-rates-tutorial/source_exchange_rates_tutorial/schemas/rates.json`
 
-You can download the JSON file describing the output schema with all currencies [here](https://raw.githubusercontent.com/airbytehq/airbyte/master/airbyte-cdk/python/docs/tutorials/http_api_source_assets/exchange_rates.json) for convenience and place it in `schemas/`.
+You can download the JSON file describing the output schema with all currencies [here](../../tutorials/cdk-tutorial-python-http/exchange_rates_schema.json) for convenience and place it in `schemas/`.
```
Comment (girarda) [inline docs/connector-development/config-based/tutorial/4-reading-data.md:29]: point to this file docs/connector-development/tutorials/cdk-tutorial-python-http/exchange_rates_schema.json

File: docs/connector-development/config-based/tutorial/4-reading-data.md [hunk 2]
```diff
@@ -26,10 +26,10 @@ See the [catalog guide](https://docs.airbyte.com/understanding-airbyte/beginners
...
 
 ```bash
-curl https://raw.githubusercontent.com/airbytehq/airbyte/master/airbyte-cdk/python/docs/tutorials/http_api_source_assets/exchange_rates.json > source_exchange_rates_tutorial/schemas/rates.json
+curl https://raw.githubusercontent.com/airbytehq/airbyte/master/docs/connector-development/tutorials/cdk-tutorial-python-http/exchange_rates_schema.json > source_exchange_rates_tutorial/schemas/rates.json
```
Comment (girarda) [inline docs/connector-development/config-based/tutorial/4-reading-data.md:32]: point to this file docs/connector-development/tutorials/cdk-tutorial-python-http/exchange_rates_schema.json

It's not on master yet, but you can replace `master` with this commit hash`a1f1aa952d89b6c05bbd9d799758e4501cc4ac80`:
  `curl https://raw.githubusercontent.com/airbytehq/airbyte/a1f1aa952d89b6c05bbd9d799758e4501cc4ac80/docs/connector-development/tutorials/cdk-tutorial-python-http/exchange_rates_schema.json`

File: docs/connector-development/tutorials/cdk-tutorial-python-http/configured_catalog.json [hunk 3]
```diff
@@ -0,0 +1,134 @@
+{
```
Comment (girarda) [inline docs/connector-development/tutorials/cdk-tutorial-python-http/configured_catalog.json:1]: pasted from https://github.com/airbytehq/airbyte/commit/514be64337debbd2a355ea6ea5d667757e1f999c#diff-fbfe01b91b465422496095c6936a30e3f695ce16429dc07bde52dff80c83d916

**Review by sophia-wiley — APPROVED** [review]

**Comment by sophia-wiley** [comment]
Thank you!!

**Comment by davydov-d** [comment]
hey @girarda this PR fails Airbyte CDK build, can you please take a look? https://github.com/airbytehq/airbyte/actions/runs/4429520944/jobs/7770587160

**Pull request merged**